infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PerWorkspacePVCStrategyTest.java

```diff
@@ -108,11 +109,11 @@
   @Mock private Workspace workspace;
   @Mock private EphemeralWorkspaceAdapter ephemeralWorkspaceAdapter;
 
-  private PerWorkspacePVCStrategy perWorkspaceStratefy;
+  private PerWorkspacePVCStrategy perWorkspaceStrategy;
```

Comment:
From the way it currently works, I don't think so. The only changes to the common/unique strategies is that they have to use `podData` instead of `pod` to catch `Deployments`. This strategy just uses workspace id AFAICT.

Reply:
ok